bin/GuiConfigs/Nekotekina by GooseWing.qss

```diff
@@ -437,13 +462,19 @@ QLabel#log_stack {
 
 /* Memory Viewer */
 QLabel#memory_viewer_address_panel {
-	color: #0000ff; /* Font Color: Blue */
+	color: #ffd785; /* Font Color: Light */
+	border: 0.05em solid #ffd785;
+	background: rgba(52, 49, 40, 0.9);
 }
 QLabel#memory_viewer_hex_panel {
-	color: #4d4940; /* Font Color: Grey */
+	color: #bfa163; /* Font Color: Grey */
+	border: 0.05em solid #ffd785;
+	background: rgba(52, 49, 40, 0.9);
 }
 QLabel#memory_viewer_ascii_panel {
-	color: #4d4940; /* Font Color: Grey */
+	color: #bfa163; /* Font Color: Grey */
+	border: 0.05em solid #ffd785;
+	background: rgba(52, 49, 40, 0.9);

```

Comment:
these three need:
`font-family: none;`
Otherwise the font in the memory viewer panel is not monospace, as intended.